action.yml

```diff
@@ -60,6 +60,73 @@ inputs:
       setup-gcloud action with exported credentials.
     required: false
 
+  source:
+    description: |-
+      The location of the source to build. The location can be a directory on
+      a local disk or a gzipped archive file (.tar.gz) in Google Cloud Storage.
+    required: false
+
+  suffix:
+    description: |-
+      Specify the suffix of the revision name. Revision names always start with
+      the service name automatically. For example, specifying 'v1' for a service
+      named 'helloworld', would lead to a revision named 'helloworld-v1'.
+    required: false
+
+  tag:
+    description: |-
+      Traffic tag to assign to the newly created revision.
+    required: false
+
+  no_traffic:
+    description: |-
+      True to avoid sending traffic to the revision being deployed. Setting
+      this flag assigns any traffic assigned to the LATEST revision to the 
+      specific revision bound to LATEST before the deployment. The effect is 
+      that the revision being deployed will not receive traffic. After a 
+      deployment with this flag the LATEST revision will not receive traffic 
+      on future deployments. 
+    required: false
+
+  revision_traffic:
+    description: |-
+      Comma separated list of traffic assignments in the form REVISION-NAME=PERCENTAGE.
+      REVISION-NAME must be the name for a revision for the service as returned
+      by 'gcloud beta run list revisions'. PERCENTAGE must be an integer 
+      percentage between 0 and 100 inclusive. 
+      Ex service-nw9hs=10,service-nw9hs=20 Up to 100 percent of traffic may be
+      assigned. If 100 percent of traffic is assigned, the Service traffic is
+      updated as specified. If under 100 percent of traffic is assigned, the 
+      Service traffic is updated as specified for revisions with assignments and 
+      traffic is scaled up or down down proportionally as needed for revision 
+      that are currently serving traffic but that do not have new assignments.
+    required: false
+
+  tag_traffic:
+    description: |-
+      Comma separated list of traffic assignments in the form TAG=PERCENTAGE.
+      TAG must match a traffic tag on a revision of the service. PERCENTAGE must
+      be an integer percentage between 0 and 100 inclusive. Up to 100 percent of
+      traffic may be assigned. If 100 percent of traffic is assigned, the service
+      traffic is updated as specified. If under 100 percent of traffic is 
+      assigned, the service traffic is updated as specified to the given tags,
+      and other traffic is scaled up or down proportionally.
+    required: false
+
+  flags:
```

Comment:
Should maybe some of the other params like `--no-traffic` be condensed into this? 

Also would there be a case where the flag may contain a space?

Reply:
I like surfacing this set of flags functionality (traffic management, version tagging) explicitly through the inputs. I've updated the flag splitting to not split if in quotes with tests.